<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceEditSheet</class>
 <widget class="QWidget" name="FaceEditSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Edit Sheet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-300</y>
        <width>355</width>
        <height>686</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Material</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Texture Scale</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="textureScaleXSpin">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.250000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="textureScaleYSpin">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.250000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Align</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnAlignWorld">
               <property name="text">
                <string>World</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAlignFace">
               <property name="text">
                <string>Face</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Texture Shift</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="textureShiftXSpin">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="textureShiftYSpin">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Justify</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="2">
              <widget class="QPushButton" name="btnJustifyRight">
               <property name="text">
                <string>Right</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btnJustifyTop">
               <property name="text">
                <string>Top</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="btnJustifyCenter">
               <property name="text">
                <string>Center</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btnJustifyBottom">
               <property name="text">
                <string>Bottom</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="btnJustifyLeft">
               <property name="text">
                <string>Left</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btnFit">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Fit</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="chkTreatAsOne">
               <property name="text">
                <string>Treat as one</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Rotation</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QDoubleSpinBox" name="rotationSpin">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="wrapping">
                <bool>true</bool>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Current Material</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="materialIcon">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="materialFileEdit"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnReplace">
                 <property name="text">
                  <string>Replace...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnBrowse">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
